packages/react-core/src/demos/examples/AlertGroup/AlertGroupToastWithNotificationDrawer.tsx

```diff
@@ -0,0 +1,365 @@
+              id="dropdown-toggle-id-0"
+            />
+          }
+          isOpen={openDropdownKey === 'dropdown-toggle-id-0'}
+          isPlain
+          dropdownItems={notificationDrawerActions}
+          id="notification-drawer-0"
+          position={DropdownPosition.right}
+        />
+      </NotificationDrawerHeader>
+      <NotificationDrawerBody>
+        {notifications.length !== 0 && (
+          <NotificationDrawerList>
+            {notifications.map(({ key, variant, title, srTitle, description, timestamp }) => (
+              <NotificationDrawerListItem key={key} variant={variant} isRead={isNotificationRead(key)}>
+                <NotificationDrawerListItemHeader variant={variant} title={title} srTitle={srTitle}>
+                  <Dropdown
+                    position={DropdownPosition.right}
+                    onSelect={onDropdownSelect}
+                    toggle={<KebabToggle onToggle={isActive => onDropdownToggle(key, isActive)} id={key.toString()} />}
+                    isOpen={openDropdownKey === key}
+                    isPlain
+                    dropdownItems={notificationDrawerDropdownItems(key)}
+                    id={key.toString()}
+                  />
+                </NotificationDrawerListItemHeader>
+                <NotificationDrawerListItemBody timestamp={timestamp}> {description} </NotificationDrawerListItemBody>
+              </NotificationDrawerListItem>
+            ))}
+          </NotificationDrawerList>
+        )}
+        {notifications.length === 0 && (
+          <EmptyState variant={EmptyStateVariant.full}>
+            <EmptyStateIcon icon={SearchIcon} />
+            <Title headingLevel="h2" size="lg">
+              No notifications found
+            </Title>
+            <EmptyStateBody>There are currently no notifications.</EmptyStateBody>
+          </EmptyState>
+        )}
+      </NotificationDrawerBody>
+    </NotificationDrawer>
+  );
+
+  return (
+    <DashboardWrapper
+      header={<DashboardHeader notificationBadge={notificationBadge} />}
+      mainContainerId="main-content-page-layout-default-nav"
+      notificationDrawer={notificationDrawer}
+      isNotificationDrawerExpanded={isDrawerExpanded}
+    >
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">Alert Group with Notification Drawer demo</Text>
+          <Text component="p">
+            New alerts can be added with buttons below. Each alert has a timeout of 7 seconds, however, even after the
+            timeout expires, all alerts are still visible in the notification drawer. By default, only 3 alerts are
+            displayed. The rest can be accessed in the notification drawer after clicking on the bell icon in the header
+            or by clicking on the overflow message.
+          </Text>
+        </TextContent>
+      </PageSection>
+
+      <PageSection variant={PageSectionVariants.light}>
+        <button
```

Comment:
Just curious why you don't use `<Button />` here instead of adding the classes manually?

Reply:
That's a good point, changed to `<Button>`